app_dev.yaml

```diff
@@ -247,6 +247,10 @@ libraries:
 
 env_variables:
   PYTHONHTTPSVERIFY: 1
+  GAE_USE_SOCKETS_HTTPLIB : "anyvalue"
```

Comment:
Btw where's the official documentation for this, could you add it as a comment above this line? "anyvalue" just sounds weird and devs will probably wonder about it :)

Reply:
Done.